### ISSUE TITLE
API error MongoError:

### ISSUE THREAD

**Issue by ersah123** [cezerin/cezerin#401]
Hi,
I have just installed this by using this documentation:
https://github.com/cezerin/cezerin/blob/master/docs/getting-started.md
`npm start`

Then i execute all the queries here:
https://github.com/cezerin/cezerin/blob/master/docs/initialize-mongodb.md

When i navigate to http://localhost:3000/admin/products, i'm getting following error from request:

```
error: API error MongoError: The top-level _id field is the only field currently supported for exclusion
    at queryCallback (/home/juggernaut/Desktop/e-commerce/cezerin/node_modules/mongodb-core/lib/cursor.js:223:25)
    at /home/juggernaut/Desktop/e-commerce/cezerin/node_modules/mongodb-core/lib/connection/pool.js:541:18
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
```
MongoDB shell version: 3.2.19
Node v8.4.0
Ubuntu 16.04

**Comment by ersah123** [comment]
src/api/server/services/products/products.js

When i remove this part, error disappears:
```
    itemsAggregation.push({ $project: {
      "categories.description": 0,
      "categories.meta_description": 0,
      "categories._id": 0,
      "categories.date_created": 0,
      "categories.date_updated": 0,
      "categories.image": 0,
      "categories.meta_title": 0,
      "categories.enabled": 0,
      "categories.sort": 0,
      "categories.parent_id": 0,
      "categories.position": 0
    }});
```
I will take a look tomorrow again.

**Comment by sonufrienko** [comment]
@ersah123 
I see this error. Fixed by upgrade MongoDB to 3.4.
Try to run MongoDB from Docker.

**Comment by ersah123** [comment]
Thank you, using MongoDB from docker fixes the problem.
